Remove link to nonexistent VSCode Extension (#6858)
dotnet/core: Documentation/microsoft-team.md

> The tools make it easier to use open source and participate in open source projects:

Reviewer: ```suggestion
The following tool makes it easier to use open source and participate in open-source projects:
```

* [Browser Extension](https://docs.opensource.microsoft.com/tools/browser.html) -- Identifies Microsoft employees on GitHub.

> The browser extension is recommended.

Reviewer: I'd also remove this since the heading already says it's recommended.

```suggestion
```

> 

Reviewer: ```suggestion
```
